components/enrichers/custom-annotation/task.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: enricher-custom-annotation
+  labels:
+    v1.dracon.ocurity.com/component: enricher
+spec:
+  description: Identifies a code owner for each finding.
+  params:
+  - name: enricher-custom-annotation-annotation
```

Comment:
🤔 this is a bit confusing to read, should we rename it to something easier to glance to?

Perhaps `enricher-custom-annotation-base-annotation`